card/lib/card/query/join.rb

```diff
@@ -8,13 +8,16 @@ class Join
                     :from_field, :to_field,
                     :superjoin, :subjoins
 
+      # documentation??? -pk
       def initialize opts={}
         from_and_to opts
         opts.each do |key, value|
           send "#{key}=", value if value.present?
         end
         @from_field ||= :id
         @to_field   ||= :id
+
+        # why set to empty? why not allow to initialize with conditions? -pk
```

Comment:
I doubt this was explicitly done this way to prevent initializing with subjoins; I don't think it came up.  If using `||=` doesn't break any tests, I would expect it's safe.